cypress/e2e/article.cy.js

```diff
@@ -3,22 +3,45 @@
 
 describe('Article', () => {
   before(() => {
-
+    // Before running any test, ensure you are logged in
+    cy.login('testuser@example.com', 'password'); // Modify based on your actual login command
   });
 
   beforeEach(() => {
-    cy.task('db:clear');
+    cy.task('db:clear'); // Clear database before each test
```

Comment:
The task 'db:clear' might not be defined in your Cypress setup. Ensure that you have a task defined in your Cypress plugins file to handle 'db:clear'.